apps/test-stress-can/src/main.cpp

```diff
@@ -0,0 +1,38 @@
+#include "mbed.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+
+CANBus can(CAN_RX, CAN_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+CANMsg msg;
+
+DigitalOut led(LED1);
+InterruptIn btn(BUTTON_1);
+
+constexpr uint16_t sent = 1000;
+uint16_t received = 0;
+ 
+// create an event queue
+EventQueue queue;
+
+void receiveMessageOutside(CANMsg &msg) {
+  // process msg or something
+  received++;
+}
+ 
+void receiveMessageInside() {
+  CANMsg msg;
+  if (can.read(msg)){
+    queue.call(&receiveMessageOutside, msg);
+  }
```

Comment:
else { MBED_ASSERT(false); }
add the above. if the program ever gets to that line it will stop. it should never get to that line

Reply:
this is just something we should do for this test app and not for the real deal

Reply:
i think we should probably print some error here with some FATAL THIS SHOULD NEVER HAPPEN msg. idk if we have any plan for using mbed error handlers https://os.mbed.com/docs/mbed-os/v6.6/apis/error-handling.html#:~:text=The%20error%20handling%20system%20in,captured%20includes%20the%20following%20information%3A&text=Thread%20entry%20address%20%E2%80%93%20Entry%20function,is%20built%20with%20RTOS%20support.

Reply:
we'll think abt this once someone does logger